fala/apps/adviser/tests/page_objects.py

```diff
@@ -0,0 +1,75 @@
+class FalaPage(object):
+    def __init__(self, page):
+        self._page = page
+
+    @property
+    def h1(self):
+        return self._page.locator("h1")
+
+    @property
+    def language_dropdown(self):
+        return self._page.locator('select[class="goog-te-combo"]')
+
+    def item_from_text(self, text):
+        return self._page.get_by_text(text)
+
+    def select_specific_page(self, page_no):
+        return self._page.locator(f'a.govuk-pagination__link:has-text(" {page_no} ")')
+
+    def change_search(self):
```

Comment:
@HettieS I think the reason we have the `change_search` method here is that we have a `change search` button on both the 'results page' and the 'other region page'. 

We want to return the SearchPage object as some tests want to use the methods in that object within test.

Reply:
Yeah I can see it is used in a couple places. I think it is probably just my (limited) understanding of Python making this look weird to me. I thought we were sort of forcing the output to be a search page (rather than letting the application render whatever comes after the click) but on reflection I guess the test would fail if something unexpected happened